meinberlin/apps/votes/forms.py

```diff
@@ -23,21 +23,20 @@ def __init__(self, placeholder=None, *args, **kwargs):
     }
 
     def clean(self, value):
-
         # ensure no spaces or dashes
         value = value.replace(" ", "").replace("-", "")
 
         # check the value is not too short or too long
-        if len(value) < 12:
+        if len(value) < 16:
             raise forms.ValidationError(self.error_messages["invalid_short"])
-        elif len(value) > 12:
+        elif len(value) > 19:
```

Comment:
Shouldn't this also be 16 as the dashes and spaces are already removed?

Reply:
yes, good catch!